test/src/tests-identities-attributes.ts

```diff
@@ -628,8 +721,11 @@ describe('identities and attributes', function() {
         fetchMock.resetHistory();
         mParticle.Identity.getCurrentUser().setUserAttribute('age', '25');
 
-        let body = JSON.parse(fetchMock.lastOptions().body)
-        body.user_attributes.should.have.property('age', '25')
+        let body: UserAttributes = JSON.parse(
+            `${fetchMock.lastOptions().body}`
+        ) as UserAttributes; // Convert object to string
```

Comment:
Nit - I think this comment should go above this code block, rather than being in line with the TS `as` declaration since the JSON.parse extends 3 lines

Reply:
also, is the reason you are using interpolation because TS doesn't recognize that this `fetchMock.lastOptions().body` is already a string?

Reply:
I'm actually going to rework this to be a bit more clear as to why we're parsing the body as a string and so we can maintain the correct type of the subsequent Event.

TS actually recognizes that `fetchMock.lastOptions().body` is an Object, but doesn't like that we're trying to parse it as if it were a string in this line because TS expects and object. I personally find using string interpolation to be more readable than using `as unknown as string`, but I can use that if you think it's more clear.